galette/lib/Galette/Entity/Adherent.php

```diff
@@ -857,9 +858,60 @@ public function getDues(): string
                 $ret = _T("No longer member");
             }
         }
+
+        if (self::getCountDonations() > 0) {
```

Comment:
Method is not static:
```suggestion
        if ($this->getCountDonations() > 0) {
```